packages/manager/apps/ips/src/pages/listing/components/IpDatagrid/IpDatagrid.spec.tsx

```diff
@@ -109,4 +118,26 @@ describe('IpDatagrid Component', async () => {
       expect(queryByText('239.99.279.206/32')).toBeNull();
     });
   });
+
+  it('should display expandable rows for ip group', async () => {
+    useGetIpListMock.mockReturnValue({
+      ipList,
+      isLoading: false,
+      error: undefined,
+    });
+    setIpToSearch('241.94.186.48');
+    const { container, getByText } = renderComponent();
+    const expandButton = await getButtonByIcon({
+      container,
+      iconName: ODS_ICON_NAME.chevronRight,
+    });
+
+    await act(() => {
+      fireEvent.click(expandButton);
+    });
+
+    await waitFor(() => {
+      expect(getByText('row-241.94.186.48/28')).toBeDefined();
```

Comment:
Improvement: Not sure if it add more value. 

Can we getByText inside uiTestHelpers ?